description: Disables JavaScript dialog boxes that prompt when one tries to close a page

--- a/components/app_modal/javascript_dialog_manager.cc
+++ b/components/app_modal/javascript_dialog_manager.cc
@@ -128,28 +128,7 @@ void JavaScriptDialogManager::RunBeforeUnloadDialog(
     const base::string16& message_text,
     bool is_reload,
     const DialogClosedCallback& callback) {
-  const base::string16 title = l10n_util::GetStringUTF16(is_reload ?
-      IDS_BEFORERELOAD_MESSAGEBOX_TITLE : IDS_BEFOREUNLOAD_MESSAGEBOX_TITLE);
-  const base::string16 footer = l10n_util::GetStringUTF16(is_reload ?
-      IDS_BEFORERELOAD_MESSAGEBOX_FOOTER : IDS_BEFOREUNLOAD_MESSAGEBOX_FOOTER);
-
-  base::string16 full_message =
-      message_text + base::ASCIIToUTF16("\n\n") + footer;
-
-  extensions_client_->OnDialogOpened(web_contents);
-
-  AppModalDialogQueue::GetInstance()->AddDialog(new JavaScriptAppModalDialog(
-      web_contents,
-      &javascript_dialog_extra_data_,
-      title,
-      content::JAVASCRIPT_MESSAGE_TYPE_CONFIRM,
-      full_message,
-      base::string16(),  // default_prompt_text
-      false,       // display_suppress_checkbox
-      true,        // is_before_unload_dialog
-      is_reload,
-      base::Bind(&JavaScriptDialogManager::OnDialogClosed,
-                 base::Unretained(this), web_contents, callback)));
+  callback.Run(true, base::ASCIIToUTF16("bye-bye"));
 }
 
 bool JavaScriptDialogManager::HandleJavaScriptDialog(

